### ISSUE TITLE
Code Review - Adrian

### ISSUE THREAD

**Issue by a-terlik** [eazzyone11/OOP-#2]
### **Positives**
- Code runs well
- Includes testing and validation
- Most variables are well named

### **Improvements**
- Code needs comments as it can be really hard to follow
- Naming convention of variables should be followed more closely, as some are inaccurately named
- The code could be abstracted in parts as you have used 5 separate files when 4 could be sufficient

TL;DR: The code quality is quite high and does what it should, however the code should be commented and set out a bit more clearly to make it easier to follow

**Comment by eazzyone11** [comment]
![image](https://user-images.githubusercontent.com/127356300/224059218-d05c918e-b89c-4228-96c7-b0122440b67f.png)
more comments have been added to the code as requested
